### PR TITLE
Exclude DecimalFormat.setGroupingSize from openj9 jdk15

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
- Exclude for https://github.com/eclipse/openj9/issues/8620

Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Tested: Test_grinder/job/Grinder/6232/console

**Comment by Mesbah-Alam** [comment]
@JasonFengJ9 - please review.

File: openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MauveMultiThreadLoadTest.java [hunk 1]
```diff
@@ -60,11 +60,12 @@ public void execute(StfCoreExtension test) throws StfException {
...
 		}
 		
-		/* Temporarily exclude DecimalFormat.setGroupingSize from openj9 jdk14 
+		/* Temporarily exclude DecimalFormat.setGroupingSize from openj9 jdk14 and jdk15
 			Ref: https://github.com/eclipse/openj9/issues/8086
+				 https://github.com/eclipse/openj9/issues/8620
 		 */
 		if (jvm.isIBMJvm() 
-			&& jvm.getJavaVersion() == 14) {
+			&& (jvm.getJavaVersion() == 14 || jvm.getJavaVersion() == 15)) {
```
Comment (JasonFengJ9) [inline openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MauveMultiThreadLoadTest.java]: How about `jvm.getJavaVersion() >= 14` to exclude future versions?

**Comment by Mesbah-Alam** [comment]
Sure. Update PR.

File: openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MauveMultiThreadLoadTest.java (hunk 1 above)
Comment (Mesbah-Alam) [reply openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MauveMultiThreadLoadTest.java]: Sure. Updated PR.

**Comment by Mesbah-Alam** [comment]
@lumpfish - I'll make a separate PR to remove isIBMJvm check.

**Pull request merged**